src/main/host/shimipc.c

```diff
@@ -10,12 +10,12 @@
 
 // We use an int here because the option parsing library doesn't provide a way
 // to set a boolean flag to false explicitly.
-static gint _sendExplicitBlockMessage = true;
+static gint _useExplicitBlockMessage = true;
 OPTION_EXPERIMENTAL_ENTRY(
-    "send-explicit-block-message", 0, 0, G_OPTION_ARG_INT, &_sendExplicitBlockMessage,
+    "send-explicit-block-message", 0, 0, G_OPTION_ARG_INT, &_useExplicitBlockMessage,
```

Comment:
Replace the 0 in the arg preceding `G_OPTION_ARG_INT` to `G_OPTION_FLAG_NONE`. (That makes it more obvious which arg to change if we want to change it to `G_OPTION_FLAG_REVERSE`  later.)

Reply:
Added